src/rshim.c

```diff
@@ -2725,17 +2728,21 @@ static int rshim_load_cfg(void)
     } else if (!strcmp(key, "DROP_MODE")) {
       rshim_drop_mode = (atoi(value) > 0) ? 1 : 0;
       continue;
-    } else if (!strcmp(key, "PCIE_RESET_DELAY")) {
-      rshim_pcie_reset_delay = atoi(value);
-      continue;
     } else if (!strcmp(key, "USB_RESET_DELAY")) {
       rshim_usb_reset_delay = atoi(value);
       continue;
-#ifdef HAVE_RSHIM_PCIE
```

Comment:
is the HAVE_RSHIM_PCIE flag not needed anymore? if so, could remove it from the Makefile also

Reply:
This HAVE_RSHIM_PCIE is still needed to compile image for BMC which doesn't have the pcie library. 
 (./configure --enable-pcie=no).  The HAVE_RSHIM_PCIE is not needed here is because I moved rshim_pcie_intr_poll_interval from rshim_pcie.c to rshim.c to put all the rshim.conf configuration variable together. This is just for code cleanup.  Another way is to put the rshim_pcie_xxx variables in rshim_pcie.c, but then we'll need to add HAVE_RSHIM_PCIE in various places in order to compile for "./configure --enable-pcie=no".